### PR TITLE
[Merged by Bors] - feat: modify `AEEqFun.cast` so that it chooses constant functions

### PR DESCRIPTION
**Pull request by fpvandoorn**
---

[Open in Gitpod](https://gitpod.io/from-referrer/)

### CODE REVIEW

File: Mathlib/MeasureTheory/Function/AEEqFun.lean [hunk 1]
```diff
@@ -121,18 +121,27 @@ def mk {β : Type*} [TopologicalSpace β] (f : α → β) (hf : AEStronglyMeasur
   Quotient.mk'' ⟨f, hf⟩
 #align measure_theory.ae_eq_fun.mk MeasureTheory.AEEqFun.mk
 
+/-- Auxiliary condition used in the cast function. -/
+def cast.Cond (f : α →ₘ[μ] β) : Prop :=
+  ∃ (g : α → β) (hg : StronglyMeasurable g),
+    f = mk g hg.aestronglyMeasurable ∧ g ∈ range (Function.const α)
```
Comment (ADedecker) [inline Mathlib/MeasureTheory/Function/AEEqFun.lean:127]: Why not use `∃ (b : β), f = mk (Function.const α b) ...` ?

Comment (fpvandoorn) [reply Mathlib/MeasureTheory/Function/AEEqFun.lean:127]: I could do that, originally I was planning to replace `g ∈ range (Function.const α)` with `g ∈ range (Function.const α) \or Continuous g`.

Comment (urkud) [reply Mathlib/MeasureTheory/Function/AEEqFun.lean:127]: Please do that.

**Comment by urkud** [comment]
Thanks! :tada:
bors merge

**Comment by urkud** [comment]
Please fix CI, then send it to bors
bors r-
bors d+

File: Mathlib/MeasureTheory/Function/AEEqFun.lean [hunk 2]
```diff
@@ -111,23 +111,29 @@ def mk {β : Type*} [TopologicalSpace β] (f : α → β) (hf : AEStronglyMeasur
   Quotient.mk'' ⟨f, hf⟩
 
 /-- Coercion from a space of equivalence classes of almost everywhere strongly measurable
```
Comment (urkud) [inline Mathlib/MeasureTheory/Function/AEEqFun.lean:113]:
```suggestion
open scoped Classical in
/-- Coercion from a space of equivalence classes of almost everywhere strongly measurable
```

**Comment by urkud** [comment]
bors merge

**Pull request closed without merging**